tasks/tests/test_edit_tasks.py

```diff
@@ -0,0 +1,16 @@
+from tasks.models import Status, Priority, Task
+import pytest
+
+
+@pytest.mark.django_db
+class TestEditTasks:
+
+    def test_update_status(self, task_1):
+        task_1.update_status(Status.DONE)
+        assert task_1.status == Status.DONE
```

Comment:
maybe it will be better to add assert of the previous status like:
```suggestion
        assert task_1.status == Status.IN_PROGRESS
        task_1.update_status(Status.DONE)
        assert task_1.status == Status.DONE
```
in this way we will be sure that the update happened

Reply:
I will add this, but pay attention that we already made tests to check create task. As a result creating task with Status.IN_PROGRESS works, so now checking it again is just adding unnecessary check for the test.
In tests you always look about what you are testing - here I am testing that the status is changed not the the status is correct.

Reply:
I saw that you already add the assert before the update, thank you!!
just to make the point more clear:
from my POV when you test "update" of any field it is better to make sure that you have the desired status before you start the test("update") because sometimes someone else can edit the code and change the fixture to set the status as  DONE for any reason that he needs and you in your test you have only the following 2 lines:
        task_1.update_status(Status.DONE)
        assert task_1.status == Status.DONE
It will keep succeeding even though nothing was updated.